changelogs/unreleased/bump-libcurl-to-7.83.0.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
```

Comment:
In my understanding we should target this PR to `master` and cherry-pick to `2.10` then.

Reply:
Updated.